lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -122,6 +122,21 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
       element.appendChild(this.renderCategoryHeader(category, groups));
     }
 
+    // Budgets
+    const budgetAudit = category.auditRefs.find((audit) => audit.id === 'resource-budget');
+    if (budgetAudit && budgetAudit.result && budgetAudit.result.details) {
+      const budgetsGroupEl = this.renderAuditGroup(groups['budgets']);
+      const tmpl = this.dom.cloneTemplate('#tmpl-lh-opportunity-header', this.templateContext);
+      budgetsGroupEl.appendChild(this.dom.find('.lh-load-opportunity__header', tmpl));
```

Comment:
can you make a new header template (even if it's just a copy with new budget class names)

Reply:
agree with the others but i dont think its necessary to maek a new header template. seems okay to reuse this one.

Reply:
> agree with the others but i dont think its necessary to maek a new header template. seems okay to reuse this one.

booooooo 😛 

It's 300 bytes. We can afford them in exchange for avoiding the question "why does the budget section have a `lh-load-opportunity__header` className? :)

Make it a two-column performance header and then we can talk